### PR TITLE
Updated SuccubusAi

### PR DESCRIPTION
**Pull request by Sairii**
Updated the MagicStack Attack and Prepares.

### CODE REVIEW

File: system/scripts/ai/succubus.cs [hunk 1]
```diff
@@ -683,21 +672,10 @@ protected override IEnumerable Aggro()
...
-						Do(Wait(500));
-					}
-					else
-					{
-						Do(PrepareSkill(SkillId.Lightningbolt)); // 2 charge
-						Do(Wait(1000, 2000));
-						Do(Attack(1, 4000));
-						Do(Wait(500));
-					}
+					Do(PrepareSkill(SkillId.Lightningbolt, Rnd(1, 1, 2)));
```
Comment (Sairii) [inline system/scripts/ai/succubus.cs]: Pretty much the same thing. I want that wait and with stackattack I dont have that.

File: system/scripts/ai/succubus.cs [hunk 2]
```diff
@@ -399,7 +388,7 @@ protected override IEnumerable Alert()
 
 			if (Random() < 30)
 			{
-				Do(PrepareSkill(SkillId.Firebolt)); // Actually does only attack with previous charges, charges only if no charges
```
Comment (exectails) [inline system/scripts/ai/succubus.cs]: StackAttack does have a builtin 1-2 seconds wait time.

Comment (Sairii) [inline system/scripts/ai/succubus.cs]: actually no im wrong, i can do the wait before the stack attack

**Comment by exectails** [comment]
Thanks for the update.

**Pull request merged**